specification/monitor/data-plane/Microsoft.Insights/preview/2023-03-01-preview/metricBatch.json

```diff
@@ -0,0 +1,484 @@
+      ]
+    },
+    "AdditionalInfoErrorResponse": {
+      "description": "The response to a metrics query that results in a bad request, with optional additional information.",
+      "type": "object",
+      "properties": {
+        "additionalInfo": {
+          "description": "Additional information about the error",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "type": {
+                "description": "The type of the info property (e.g. string).",
+                "type": "string"
+              },
+              "info": {
+                "description": "Additional information related to the error.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "code": {
+          "description": "Error code",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string"
+        }
+      }
+    },
+    "MetricResultsResponse": {
+      "description": "The metrics result for a resource.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "timespan": {
+            "type": "string",
+            "description": "The timespan for which the data was retrieved. Its value consists of two datetimes concatenated, separated by '/'.  This may be adjusted in the future and returned back from what was originally requested."
+          },
+          "interval": {
+            "type": "string",
+            "format": "duration",
+            "description": "The interval (window size) for which the metric data was returned in.  This may be adjusted in the future and returned back from what was originally requested.  This is not present if a metadata request was made."
+          },
+          "namespace": {
+            "type": "string",
+            "description": "The namespace of the metrics been queried"
+          },
+          "resourceregion": {
+            "type": "string",
+            "description": "The region of the resource been queried for metrics."
+          },
+          "resourceid": {
+            "type": "string",
+            "description": "The resource that has been queried for metrics."
+          },
+          "value": {
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/Metric"
+            },
+            "description": "the value of the collection."
+          }
+        },
+        "required": [
+          "timespan",
+          "value"
+        ]
+      }
+    },
+    "LocalizableString": {
+      "type": "object",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "type": "string",
+          "description": "the invariant value."
+        },
+        "localizedValue": {
+          "type": "string",
+          "description": "the locale specific value."
```

Comment:
How is the locale specified? I don't see any parameter on the endpoint that obviously answers this.